dashboard/app/views/levels/editors/_dsl.html.haml

```diff
@@ -27,7 +27,7 @@
 .script_text
   = text_area_tag('level[dsl_text]', @level.dsl_text || (@level.dsl_default))
 
-- if [ContractMatch, Match, Multi, TextMatch, External].include? @level.class
+- if [ContractMatch, Match, Multi, TextMatch, External, LevelGroup].include? @level.class
```

Comment:
might be worth renaming the js file to `levelbuilder_markdown` or something (someone might have thought that all dsl defined levels had markdown?).

While we are at it, it would be more meaningful if levels had a markdown_enabled? (or similar) method that was polymorphic instead of checking a list of types here.